models/management.cattle.io.roletemplate.js

```diff
@@ -185,7 +185,7 @@ const RESOURCES = [
   'ProjectMonitorGraphs',
   'ProjectNetworkPolicies',
   'ProjectRoleTemplateBindings',
-  'Prometheuses',
+  'Prometheis',
```

Comment:
api?

https://prometheus.io/docs/introduction/faq/#what-is-the-plural-of-prometheus

Reply:
`Prometheuses` is intentional - [see this comment](https://github.com/rancher/dashboard/pull/4282#issue-1006795952)